### ISSUE TITLE
Do you have a plan to support save as PSD?

### ISSUE THREAD

**Issue by gblue1223** [meltingice/psd.js#41]
It seems like, doesn't support now.

**Comment by meltingice** [comment]
That's right, it's not supported right now. Writing PSDs is pretty complex. Maybe someday, but no immediate plans.

--
Ryan LeFevre (@meltingice)
HODINKEE, Inc.
Sr. Software Engineer




On Mon, Aug 24, 2015 at 8:47 PM -0700, "gblue1223" <notifications@github.com> wrote:












It seems like, doesn't support now.
